docs/user/dashboard/url-drilldown.asciidoc

```diff
@@ -0,0 +1,229 @@
+. *Event* variables that depend on the trigger context. These variables are dynamically extracted from the interaction context when the drilldown is executed.
+
+[[values-in-preview]]
+A subtle but important difference between *context* and *event* variables is that *context* variables use real values in previews when creating a URL drilldown. 
+For example, `{{context.panel.filters}}` are previewed with the current filters that applied to a panel. 
+*Event* variables are extracted during drilldown execution from a user interaction with a panel (for example, from a pie slice that the user clicked on).
+
+Because there is no user interaction with a panel in preview, there is no interaction context to use in a preview.
+To work around this, {kib} provides a sample interaction that relies on a picked <<trigger-picker, trigger>>.
+So in a preview, you might notice that `{{event.value}}` is replaced with `{{event.value}}` instead of with a sample from your data.
+Such previews can help you make sure that the structure of your URL template is valid.
+However, to ensure that the configured URL drilldown works as expected with your data, you have to retest the drilldown through the panel after saving a dashboard.
+
+You can access the full list of variables available for the current panel and selected trigger by clicking *Add variable* in the top-right corner of a URL template input.
+
+[float]
+[[variables-reference]]
+==== Variables reference
+
+
+|===
+|Source |Variable |Description
+
+|*Global*
+|
+| Static variables that don’t change depending on the place where a URL drilldown is used or which user interaction executed the drilldown.
+
+|
+| kibanaUrl
+| {kib} base URL. Useful for creating URL drilldowns that navigate within {kib}.
+
+| *Context*
+| 
+| Variables might change depending on where the drilldown is created and used.
+
+|
+| context.panel
+| Context provided by current dashboard panel.
+
+|
+| context.panel.id
+| ID of a panel.
+
+|
+| context.panel.title
+| Title of a panel.
+
+| 
+| context.panel.filters
+| List of {kib} filters applied to a panel. +  
+_Tip: Use in combination with <<helpers, rison>> helper for internal {kib} navigations with carrying over current filters._
+
+| 
+| context.panel.query.query
+| Current query string.
+
+| 
+| context.panel.query.lang
+| Current query language.
+
+| 
+| context.panel.timeRange.from +
+context.panel.timeRange.to
+| Current time picker values. +
+_Tip: Use in combination with <<helpers, date>> helper to format date_.
+
+| 
+| context.panel.timeRange.indexPatternId +
+context.panel.timeRange.indexPatternIds 
+|Index pattern ids used by a panel. 
+
+|
+| context.panel.savedObjectId
+| ID of saved object behind a panel.
+
+| *Event*
+| 
+| Variables that depend on a trigger context. These variables are dynamically extracted from the interaction context when the drilldown is executed.
+
+| *Single click* trigger
+| 
+| Variables available when *single click* trigger is used. 
```

Comment:
Maybe 

```suggestion
| Variables available when "Single click" trigger is used. 
```

as before `"Single click"` appeared in quotes `"`.

Reply:
I like `*` more and for consistency made it `*` 👍